code/modules/mob/living/carbon/human/species.dm

```diff
@@ -432,6 +432,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		var/obj/item/thing = C.get_item_by_slot(slot_id)
 		if(thing && (!thing.species_exception || !is_type_in_list(src,thing.species_exception)))
 			C.dropItemToGround(thing)
+	if(C.dna)
+		for(var/obj/item/equipped_item in C.get_equipped_items(include_pockets = TRUE))
+			if(!(equipped_item.bodytypes_whitelist & C.dna.species.bodytype))
+				C.dropItemToGround(equipped_item)
```

Comment:
Same as first review, bodytyoes_whitelist isn't used anymore so we don't need to add all this.